arch/arm/boot/dts/overlays/cutiepi-panel-overlay.dts

```diff
@@ -0,0 +1,124 @@
+/dts-v1/;
+/plugin/;
+
+/ {
+    compatible = "brcm,bcm2711";
```

Comment:
If this overlay is truly Pi 4-specific then it would be helpful to put an entry in `overlay_map.dts` so that the firmware can refuse to load it on other platforms, e.g.:
```
	cutiepi-panel {
		bcm2711;
	};
```
See https://github.com/raspberrypi/documentation/blob/develop/documentation/asciidoc/computers/configuration/device-tree.adoc#the-overlay-map-file.

Reply:
Added in last commit.